teacher.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'person'
+class Teacher < Person
+  def initialize(specialization)
+    super(age, parent_permission: true, name: 'Unknown')
+    @specialization = specialization
+  end
+end
```

Comment:
- Kindly, Override `can_use_services` method inside teacher class to always returns true. 👍 
```
def can_use_services?
    true
end
```